domains/maine.json

```diff
@@ -7,6 +7,7 @@
     
         "record": {
             "CNAME": "codemaine.github.io"
+             "TXT": "9991ed48ef1e29bebb9213d814ebd6"
```

Comment:
no brother, not like this.
make a different file named "_github-pages-challenge-Codemaine.maine.json"
with TXT record "9991ed48ef1e29bebb9213d814ebd6"
```suggestion
```